src/OOVPADatabase/D3D8LTCG/3911.inl

```diff
@@ -905,18 +905,11 @@ OOVPA_END;
 // * D3DDevice_SetTexture
 // ******************************************************************
 //81C10000F8FF89 ...C20400
-OOVPA_NO_XREF(D3DDevice_SetTexture_4, 2024, 9)
+OOVPA_NO_XREF(D3DDevice_SetTexture_4__LTCG_eax_pTexture, 2024, 9)
```

Comment:
```suggestion
OOVPA_NO_XREF(D3DDevice_SetTexture_4__LTCG_eax_pTexture, 2024, 11)
```
There are total of 11 offset value pairs, eventually this manual count will be remove in the future.